internal/resources/provisioner/resource_provisioner.go

```diff
@@ -80,6 +86,9 @@ func resourceProvisionerCreate(ctx context.Context, d *schema.ResourceData, m in
 
 	d.SetId(provisionerResponse.Provisioner.Meta.UID)
 
+	log.Printf("Wait for %d seconds after the create the operation before fetching the state", waitTime)
+	time.Sleep(waitTime)
```

Comment:
Wonder why the wait time is required to provisioner resource alone?

Reply:
Please check the comment I have added in the L28

Reply:
It's the same case with resources like cluster-group, namespace and workspace? We don't have status field in these resources?

Reply:
In those cases the API returns a response is guaranteeing that resource is created but that's not the same case here. That's why we are seeing a provider crash! now I don't know why the API implementation is this way for provisioner alone so your question is specific to API behaviour which only the service owner can answer.